tests/mwbuilders/addition.cpp

```diff
@@ -118,7 +119,7 @@ template<int D> void testAddition() {
                 THEN("the integral is zero") {
                     double e_int = e_tree.integrate();
                     double ref_int = 0.0;
-                    REQUIRE( (e_int == Approx(ref_int)) );
+                    REQUIRE( std::abs(e_int - ref_int) < thrs );
```

Comment:
I think you can still use the `Approx` form by setting the `thrs` as its `epsilon`. Look here for details https://github.com/catchorg/Catch2/blob/master/docs/assertions.md#top